content/docs/user-guide/building/requirements.md

```diff
@@ -0,0 +1,40 @@
+# Requirements
+
+Get `pip` based and `unix` based requirements for your model.
+
+## Description
+
+`mlem` can export the requirements needed for a model using the
```

Comment:
Instead of `mlem`. here we should use MLEM

Reply:
done